packages/native-stack/src/views/NativeStackView.native.tsx

```diff
@@ -190,19 +204,31 @@ const SceneView = ({
     navigationBarTranslucent,
     navigationBarHidden,
     orientation,
-    sheetAllowedDetents = 'large',
-    sheetLargestUndimmedDetent = 'all',
+    sheetLargestUndimmedDetent = -1,
     sheetGrabberVisible = false,
     sheetCornerRadius = -1.0,
+    sheetElevation = 24,
     sheetExpandsWhenScrolledToEdge = true,
+    sheetInitialDetent = 0,
     statusBarAnimation,
     statusBarHidden,
     statusBarStyle,
     statusBarTranslucent,
     statusBarBackgroundColor,
+    unstable_sheetFooter = null,
     freezeOnBlur,
   } = options;
 
+  // This is workaround for... find the commit here
+  // We want to allow only backgroundColor setting for now
+  unstable_screenStyle = unstable_screenStyle
+    ? { backgroundColor: unstable_screenStyle.backgroundColor }
+    : null;
+
+  if (sheetAllowedDetents === 'fitToContents') {
+    sheetAllowedDetents = [-1];
+  }
```

Comment:
This won't be necessary anymore - it will be now handled internally within react-native-screens once https://github.com/software-mansion/react-native-screens/pull/2356 is merged.